Sprint-3/2-practice-tdd/count.js

```diff
@@ -1,5 +1,21 @@
 function countChar(stringOfCharacters, findCharacter) {
-  return 5
+  let characterOccurrence = "";
+  if (
```

Comment:
It's usually better to return early if inputs are invalid, it's called `early return`. Can you think of a way to implement this in your function?